addons/binding/org.openhab.binding.kostalinverter/ESH-INF/thing/Channels.xml

```diff
@@ -0,0 +1,491 @@
+	</channel-type>
+	<channel-type id="scb-event-error-count-scb">
+		<item-type>Number:Dimensionless</item-type>
+		<label>SCB errors</label>
+		<description>Number of errors reported by the smart communication board</description>
+		<category>error</category>
+		<state readOnly="true" pattern="%.0f" />
+	</channel-type>
+	<channel-type id="scb-event-warning-count-scb">
+		<item-type>Number:Dimensionless</item-type>
+		<label>SCB warnings</label>
+		<description>Number of warnings reported by the smart communication board</description>
+		<category>error</category>
+		<state readOnly="true" pattern="%.0f" />
+	</channel-type>
+	<channel-type id="statistic-autarky-day">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Autarchy rate of the day</label>
+		<description>Autarky ratio of this day</description>
+		<state readOnly="true" pattern="%.2f %unit%" />
+	</channel-type>
+	<channel-type id="statistic-autarky-month">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Autarky rate of the month</label>
+		<description>Autarky ratio of this month</description>
+		<state readOnly="true" pattern="%.2f %unit%" />
+	</channel-type>
+	<channel-type id="statistic-autarky-total">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Autarky rate in total</label>
+		<description>Autarky ratio overall</description>
+		<state readOnly="true" pattern="%.2f %unit%" />
+	</channel-type>
+	<channel-type id="statistic-autarky-year">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Autarky rate of the year</label>
+		<description>Autarky ratio of this year</description>
+		<state readOnly="true" pattern="%.2f %unit%" />
+	</channel-type>
+	<channel-type id="statistic-co2saving-day">
+		<item-type>Number:Mass</item-type>
+		<label>CO2 savings of the day</label>
+		<description>Savings in Co2 emissions today</description>
+		<category>CarbonDioxide</category>
+		<state readOnly="true" pattern="%.2f %unit%" />
+	</channel-type>
+	<channel-type id="statistic-co2saving-month">
+		<item-type>Number:Mass</item-type>
+		<label>CO2 savings of the month</label>
+		<description>Savings in Co2 emissions this month</description>
+		<category>CarbonDioxide</category>
+		<state readOnly="true" pattern="%.2f %unit%" />
+	</channel-type>
+	<channel-type id="statistic-co2saving-total">
+		<item-type>Number:Mass</item-type>
+		<label>CO2 savings in total</label>
+		<description>Savings in Co2 emissions overall</description>
+		<category>CarbonDioxide</category>
+		<state readOnly="true" pattern="%.2f %unit%" />
+	</channel-type>
+	<channel-type id="statistic-co2saving-year">
+		<item-type>Number:Mass</item-type>
+		<label>CO2 savings of the year</label>
+		<description>Savings in Co2 emissions this year</description>
+		<category>CarbonDioxide</category>
+		<state readOnly="true" pattern="%.2f %unit%" />
+	</channel-type>
+	<channel-type id="statistic-homeconsumption-day">
+		<item-type>Number:Energy</item-type>
+		<label>Home consumption (day)</label>
```

Comment:
All works in labels should be start with uppercase. Also see if you can make labels short as they are shown with the state value and then short names are best. Also see if you can create labels without using parentheses.